qiita_db/portal.py

```diff
@@ -5,13 +5,66 @@
 #
 # The full license is in the file LICENSE, distributed with this software.
 # -----------------------------------------------------------------------------
+import warnings
 
 from .sql_connection import SQLConnectionHandler
 from .util import convert_to_id
+from .exceptions import QiitaDBError, QiitaDBDuplicateError
 from qiita_core.qiita_settings import qiita_config
+from qiita_core.exceptions import IncompetentQiitaDeveloperError
 
 # Only make functions available for main portal
 if qiita_config.portal == "QIITA":
+    def create_portal(portal, desc):
```

Comment:
Thinking more, this definitely should not be an object. Unlike the other objects, this is admin-only, potentially dangerous functionality. Therefore, if you instantiate something like `Portal('EMP')` you would get access to all the functionality, which shouldn't be done for something admin-only. To me, it's safer to explicitly import the functionality needed (as in the qiita pet pull request, where only the add and remove studies functions are imported).